### ISSUE TITLE
Keyboard navigation should be deactivated when navigational elements are suppressed

### ISSUE THREAD

**Issue by greatislander** [pressbooks/pressbooks-book#219]
### Prerequisites

* [x] Can you reproduce the behavior you wish to address with [all plugins except Pressbooks deactivated](http://codex.wordpress.org/Multisite_Network_Administration#Plugins)?
* [x] Are you running the [latest version of Pressbooks](http://github.com/pressbooks/pressbooks/releases/latest/), the latest version of [Pressbooks Book](http://github.com/pressbooks/pressbooks-book/releases/latest/), the latest version of [this book theme](http://github.com/pressbooks/pressbooks-book/releases/latest/) and the [latest version of WordPress](http://codex.wordpress.org/Upgrading_WordPress)?
* [x] Are you experiencing the issue while using this book theme?
* [x] Did you [perform a cursory search](https://github.com/issues?q=+is%3Aissue+repo%3Apressbooks%2Fpressbooks-book) to see if your bug or enhancement is already listed?

### Description

See https://github.com/pressbooks/pressbooks-lti-provider/issues/7. If the `<body>` has `.no-navigation` class, [this part of the script](https://github.com/pressbooks/pressbooks-book/blob/dev/assets/src/scripts/routes/single.js#L170-L182) should not be active.

Props @beckej13820 for reporting.

### Steps to Reproduce

1. Embed in LMS.
2. Disable navigational elements.
3. Attempt to use keyboard navigation.

**Expected behavior:** Keyboard navigation should be deactivated.

**Actual behavior:** Keyboard navigation is active.

### System Information

McLuhan: 2.3.0

**Comment by greatislander** [comment]
Fixed in #221.
